state-chain/runtime/src/migrations/threshold_signature_callbacks.rs

```diff
@@ -1,11 +1,13 @@
-use crate::{BitcoinInstance, Box, EthereumInstance, PolkadotInstance, Runtime, RuntimeCall};
+use crate::{
+	BitcoinBroadcaster, BitcoinInstance, Box, EthereumBroadcaster, EthereumInstance,
+	PolkadotBroadcaster, PolkadotInstance, Runtime, RuntimeCall,
+};
```

Comment:
This file shouldn't be changed, it's a migration for a particular version - I think we can delete it instead of changing here.

Reply:
Old migration removed. 
There's a separate linear ticket for removing stale migrations. I'll leave the other pallets to be done in a separate PR

Reply:
This was added back? what happened here?

Reply:
Re-removed. Might be re-added due to incorrect merge from main : (